modin/pandas/test/test_groupby.py

```diff
@@ -376,12 +398,26 @@ def maybe_get_columns(df, by):
         eval_apply(modin_groupby, pandas_groupby, func)
 
     eval_dtypes(modin_groupby, pandas_groupby)
-    eval_general(modin_groupby, pandas_groupby, lambda df: df.first(), is_default=True)
     eval_general(
-        modin_groupby, pandas_groupby, lambda df: df.backfill(), is_default=True
+        modin_groupby,
+        pandas_groupby,
+        lambda df: df.first(),
+    )
+    eval_general(
+        modin_groupby,
+        pandas_groupby,
+        lambda df: df.backfill(),
+    )
+    eval_general(
+        modin_groupby,
+        pandas_groupby,
+        lambda df: df.bfill(),
+    )
+    eval_general(
+        modin_groupby,
+        pandas_groupby,
+        lambda df: df.idxmin(),
```

Comment:
Looks like you've made optional formatting changes. Maybe you are using outdated black?

Please roll them back.

Reply:
<img width="1128" alt="Screen Shot 2022-04-27 at 9 28 50 AM" src="https://user-images.githubusercontent.com/10712109/165567356-438bd20f-bc37-48dc-b1ce-28f3d5a0b20e.png">

The latest version of black, 22.3.0, is requiring me to reformat the lines this way when I remove `is_default`. Here's a screenshot.

Reply:
The problem is in the last comma, you don't need to put it.
Example of a more compact representation:
```bash
(modin) C:\Users\amyachev\OneDrive - Intel Corporation\Desktop\projects\modin>black --diff modin 
--- modin\pandas\test\test_groupby.py   2022-04-27 18:44:47.055066 +0000
+++ modin\pandas\test\test_groupby.py   2022-04-27 18:44:55.046452 +0000
@@ -115,13 +115,11 @@
             modin_groupby = modin_df.groupby(by=by[0], as_index=as_index)
             pandas_groupby = pandas_df.groupby(by=by[-1], as_index=as_index)

         modin_groupby_equals_pandas(modin_groupby, pandas_groupby)
         eval_ngroups(modin_groupby, pandas_groupby)
-        eval_general(
-            modin_groupby, pandas_groupby, lambda df: df.ffill()
-        )
+        eval_general(modin_groupby, pandas_groupby, lambda df: df.ffill())
         eval_general(
             modin_groupby,
             pandas_groupby,
             lambda df: df.sem(),
             modin_df_almost_equals_pandas,
would reformat modin\pandas\test\test_groupby.py

All done! ✨ 🍰 ✨
```

Reply:
The problem is in the last comma, you don't need to put it.
Example of a more compact representation:
```bash
(modin) C:\Users\amyachev\OneDrive - Intel Corporation\Desktop\projects\modin>black --diff modin 
--- modin\pandas\test\test_groupby.py   2022-04-27 18:44:47.055066 +0000
+++ modin\pandas\test\test_groupby.py   2022-04-27 18:44:55.046452 +0000
@@ -115,13 +115,11 @@
             modin_groupby = modin_df.groupby(by=by[0], as_index=as_index)
             pandas_groupby = pandas_df.groupby(by=by[-1], as_index=as_index)

         modin_groupby_equals_pandas(modin_groupby, pandas_groupby)
         eval_ngroups(modin_groupby, pandas_groupby)
-        eval_general(
-            modin_groupby, pandas_groupby, lambda df: df.ffill()
-        )
+        eval_general(modin_groupby, pandas_groupby, lambda df: df.ffill())
         eval_general(
             modin_groupby,
             pandas_groupby,
             lambda df: df.sem(),
             modin_df_almost_equals_pandas,
would reformat modin\pandas\test\test_groupby.py

All done! ✨ 🍰 ✨
```